api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/QualityImprovement2020CohortQueries.java

```diff
@@ -882,6 +882,137 @@ public CohortDefinition getMQC11DEN(String indicatorFlag) {
+
+    CohortDefinition startedART = getMQC3D1(); // A
+
+    CohortDefinition b1 =
+            getPatientsFromFichaClinicaB1OrB2("B1"); // B1
+
+    CohortDefinition b1E =
+            getPatientsFromFichaClinicaB1OrB2("B1E"); // B1E
+
+    CohortDefinition b2 =
+            getPatientsFromFichaClinicaB1OrB2("B2_12"); // B2
+
+    CohortDefinition b2E =
+            getPatientsFromFichaClinicaB1OrB2("B2E"); // B2
+
+    // CohortDefinition nutritionalClass = getPatientsWithNutritionalState();
+
+    CohortDefinition pregnant =
+            commonCohortQueries.getMohMQPatientsOnCondition(
+                    true,
+                    false,
+                    "once",
+                    hivMetadata.getMasterCardEncounterType(),
+                    commonMetadata.getPregnantConcept(),
+                    Collections.singletonList(hivMetadata.getYesConcept()),
+                    null,
+                    null);
+
+    CohortDefinition breastfeeding =
+            commonCohortQueries.getMohMQPatientsOnCondition(
+                    true,
+                    false,
+                    "once",
+                    hivMetadata.getMasterCardEncounterType(),
+                    commonMetadata.getBreastfeeding(),
+                    Collections.singletonList(hivMetadata.getYesConcept()),
+                    null,
+                    null);
+
+    CohortDefinition transferIn =
+            commonCohortQueries.getMohMQPatientsOnCondition(
+                    false,
+                    true,
+                    "once",
+                    hivMetadata.getMasterCardEncounterType(),
+                    commonMetadata.getTransferFromOtherFacilityConcept(),
+                    Collections.singletonList(hivMetadata.getYesConcept()),
+                    hivMetadata.getTypeOfPatientTransferredFrom(),
+                    Collections.singletonList(hivMetadata.getArtStatus()));
+
+    CohortDefinition transfOut = commonCohortQueries.getTranferredOutPatients();
+
+    compositionCohortDefinition.addSearch("A", EptsReportUtils.map(startedART, MAPPING));
+
+    compositionCohortDefinition.addSearch(
+            "B1", EptsReportUtils.map(b1, MAPPING));
+
+    compositionCohortDefinition.addSearch(
+            "B1E", EptsReportUtils.map(b1E, MAPPING));
+
+    compositionCohortDefinition.addSearch(
+            "B2", EptsReportUtils.map(b2, MAPPING));
+
+    compositionCohortDefinition.addSearch(
+            "B2E", EptsReportUtils.map(b2E, MAPPING));
+
+    compositionCohortDefinition.addSearch("C", EptsReportUtils.map(pregnant, MAPPING));
+
+    compositionCohortDefinition.addSearch("D", EptsReportUtils.map(breastfeeding, MAPPING));
+
+    compositionCohortDefinition.addSearch("E", EptsReportUtils.map(transferIn, MAPPING));
+
+    compositionCohortDefinition.addSearch("F", EptsReportUtils.map(transfOut, MAPPING));
+
+    if (indicatorFlag == "A" || indicatorFlag == "E" || indicatorFlag == "F")
```

Comment:
String values are compared using '==', not 'equals()', please replace **==** with **equals()**

Reply:
Do that for all where there is string comparisons, only use == for non string values